lms/djangoapps/courseware/testutils.py

```diff
@@ -71,8 +71,9 @@ def setup_course(self, default_store=None):
         with self.store.default_store(default_store):
             self.course = CourseFactory.create(**self.course_options())  # pylint: disable=attribute-defined-outside-init
             chapter = ItemFactory.create(parent=self.course, category='chapter')
+            section = ItemFactory.create(parent=chapter, category='sequential')
             self.html_block = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
-                parent=chapter,
+                parent=section,
```

Comment:
Yes.  The tests still pass.  I'm trying to remember why this change was initially made.  Perhaps I did so to add more coverage for testing sequentials.  In any case, I have added more explicit tests so I went ahead and reverted this change.